src/core/Table/Table.tsx

```diff
@@ -88,6 +92,22 @@ export type TableProps<
    * @default 300
    */
   intersectionMargin?: number;
+  /**
+   * A function that will be used for rendering a component for each row if that row is expanded.
+   * Component will be placed right after the row, in the same row group div.
```

Comment:
Update this comment. The row group part is not true anymore.

Also add the part about returning `false` or `null` if you add that functionality.